addons/l10n_bd/data/template/account.group-bd.csv

```diff
@@ -0,0 +1,32 @@
+"id","code_prefix_start","code_prefix_end","name"
+"l10n_bd_group_100100_100199","100100","100199","100100 Liquidity"
```

Comment:
I think we shouldn't have the code in the name.
The display name defined on the groups already group the prefix with the name. 
You can see it here https://github.com/odoo/odoo/blob/17.0/addons/account/models/account_account.py#L807
So you would have everything doubled